app/src/main/java/com/github/se/eduverse/ui/camera/nextScreen.kt

```diff
@@ -408,3 +426,63 @@ fun StyledButton(
         )
       }
 }
+
+fun showBottomMenu(context: Context, folderViewModel: FolderViewModel, select: (Folder) -> Unit) {
```

Comment:
I think it's better to define this function in a separate file or in the folder view model since it's not specific to the NextScreen and it's going to be needed by other components of the app